packages/web3-plugin-example/src/ERC20Token.ts

```diff
@@ -0,0 +1,141 @@
+/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+export const ERC20TokenAbi = [
```

Comment:
I think we should use existing abi + bytecode from 4x/web3.js/fixtures/build/ERC20Token.ts

Reply:
The reason a new file was created because the ABI is used [here](https://github.com/web3/web3.js/blob/3ad7e1dac7536b880d306433dbba78d360eff4dd/packages/web3-plugin-example/src/contract_method_wrappers.ts#L24) for the `ContractMethodWrappersPlugin` class, and while I think it's okay for tests to reach outside of the package directory, I don't think the package implementation code should depend on files outside of the package scope